daml-assistant/daml-helper/test/DA/Daml/Helper/Test/Ledger.hs

```diff
@@ -0,0 +1,55 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module DA.Daml.Helper.Test.Ledger (main) where
+
+import DA.Bazel.Runfiles
+import DA.Ledger.Services.PartyManagementService (PartyDetails(..))
+import DA.Ledger.Types (Party(..))
+import DA.Test.HttpJson
+import DA.Test.Sandbox
+import System.Environment.Blank
+import System.FilePath
+import System.Process
+import Test.Tasty
+import Test.Tasty.HUnit
+
+main :: IO ()
+main = do
+  setEnv "TASTY_NUM_THREADS" "1" True
+  damlHelper <-
+    locateRunfiles (mainWorkspace </> "daml-assistant" </> "daml-helper" </> exe "daml-helper")
+  defaultMain $
+    testGroup
+      "daml ledger"
+      [ withSandbox defaultSandboxConf $ \getSandboxPort -> do
+          withHttpJson getSandboxPort $ \getHttpJson ->
+            testGroup
+              "list-parties"
+              [ testCase "succeeds against HTTP JSON API" $ do
+                  HttpJson {hjPort, hjTokenFile} <- getHttpJson
+                  sandboxPort <- getSandboxPort
+                  callCommand $
+                    unwords
+                      [ damlHelper
+                      , "ledger"
+                      , "allocate-party"
+                      , "--host"
+                      , "localhost"
+                      , "--port"
+                      , show sandboxPort
+                      , "alice"
```

Comment:
It is "alice" with a lower case "a" here and "Alice" with a capital "A" in the helper library. Does that actually work?

Reply:
yes, because you issue a command with the authority actAs "Alice" from the token to allocate a party named "alice". You could just as well allocate the party "hurryabit" with the same token.

Reply:
I understand. If we want to test that the party names can indeed be different, could we please move them further apart from each other, like "Alice" and "Bob"? Right now, it looks like a typo.

Reply:
Ok, we allocate "Bob" now in the test.